SubstraitType.g4

```diff
@@ -0,0 +1,209 @@
+grammar SubstraitType;
+
+//
+fragment A : [aA];
```

Comment:
When we move this into core we should use:

```
options {
    caseInsensitive = true;
}
```

so we can eliminate all these fragments.

Reply:
I can open an issue in substrait-java for this..